### PR TITLE
Give command

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: wiki/Commands/give/index.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+There are some example commands (templates) for you.
+
+- `.give templates` shows you the list of templates.
+
+- `.give template <template id> <amount>` gives you the item in the template.
+
+## Template IDs
+
+- <b>0</b>) Knockback Stick
```
Comment (Alexander01998) [inline wiki/Commands/give/index.md]: The IDs range from 1-6, not 0-5.

**Comment by ghost** [comment]
? Sorry if wasting time

File: wiki/Commands/give/index.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+title: .give Command
+layout: wiki
+category: Command
+---
+Gives you an item with custom NBT settings in creative.
+
+## Syntax
+- `.give <item name> <amount> <metadata> <NBT data>`
```
Comment (Alexander01998) [inline wiki/Commands/give/index.md]: This syntax is outdated. See https://github.com/Wurst-Imperium/Wurst-Client/blob/master/Wurst%20Client/src/tk/wurst_client/commands/GiveCmd.java#L25

**Comment by Alexander01998** [comment]
> ? Sorry if wasting time

Nah, I'm done with Wurst 2.3.

**Comment by ghost** [comment]
? @Alexander01998

**Pull request merged**